pom.xml

```diff
@@ -57,19 +57,20 @@
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.testTarget>1.8</maven.compiler.testTarget>
 		<maven.compiler.testSource>1.8</maven.compiler.testSource>
-		<spring-cloud-build.version>3.0.0</spring-cloud-build.version>
-		<spring-cloud-commons.version>3.0.0</spring-cloud-commons.version>
-		<spring-cloud-sleuth.version>3.0.0</spring-cloud-sleuth.version>
-		<spring-cloud-gateway.version>3.0.0</spring-cloud-gateway.version>
+		<spring-cloud-build.version>3.0.1</spring-cloud-build.version>
+		<spring-cloud-commons.version>3.0.1</spring-cloud-commons.version>
+		<spring-cloud-sleuth.version>3.0.2-SNAPSHOT</spring-cloud-sleuth.version>
```

Comment:
Do we want to release a new version of Sleuth first then change everything to `3.0.2`?

Reply:
If the Wavefront changes get merged then we can release Sleuth and then make this depend on 3.0.2. Otherwise we can stay on Snapshots